db/collection_index.go

```diff
@@ -107,6 +107,26 @@ func (db *db) fetchCollectionIndexDescriptions(
 	return indexDescriptions, nil
 }
 
+func (db *db) getCollectionIndexes(
+	ctx context.Context,
+	txn datastore.Txn,
+	col client.Collection,
+) ([]CollectionIndex, error) {
+	indexDescriptions, err := db.fetchCollectionIndexDescriptions(ctx, txn, col.ID())
```

Comment:
todo: Are you sure this is necessary?  The IndexDescriptions should already be on the `col.Description()` that you are passing into this function.

Reply:
you are right. Adjusted. Nice catch!

Reply:
nice! thanks :)